projects/Mallard/src/screens/home-screen.tsx

```diff
@@ -62,6 +63,23 @@ export const HomeScreen = ({
             })),
         [demoIssues.map(({ date }) => date)],
     )
+    const issueList = issues
+        .map(issue =>
+            useIssue<Issue | undefined>(issue)({
+                success: _ => _,
+                pending: () => undefined,
+                error: _ => undefined,
```

Comment:
this feels like an anti-pattern. The purpose of this response object is to always display relevant UI. If we are eating up 2/3 of cases and passing the other straight away then this needs more rethinking

Reply:
yeah, that's what I wanted the other type for. this was quick and dirty to allow it to be used in a loop

Reply:
what is the case for an error that isn't being fulfilled by the current error type? 

I think of this three-way hook as the last stage before picking what UI to show. if we want to do other processing to the data before this happens. I don't think this is the right place for it, curious as to what cases we have that can't be handled by the backend

Reply:
100% is, but if `useResponse` was `<T, S, F, P>` we could use this in this way without this problem. But like, this was just a first shot to see if it would work.